test/spec/general.js

```diff
@@ -1,237 +1,9 @@
-                  'a0g1ZVpyVHk4UE1EbHZ4IiwiaWF0IjoxNDQ5Njk2MzMwLCJleHA' +
-                  'iOjE0NDk2OTk5MzAsImFtciI6WyJrYmEiLCJtZmEiLCJwd2QiXS' +
-                  'wianRpIjoiVFJaVDdSQ2lTeW1UczVXN1J5aDMiLCJhdXRoX3Rpb' +
-                  'WUiOjE0NDk2OTYzMzB9.YWCNE3ZvT-8ceKnAbTkmSxYE-jIPpfh' +
-                  '2s8f_hTagUUxrfdKgyWzBb9iN3GOPaQ2K6jqOFx90RI2GBzAWec' +
-                  'pel3sAxG-wvLqiy0d8g0CUb7XTHdhXOLRrXvlpbULxdNnMbBcc6' +
-                  'uOLDalBjrumOiDMLzti-Bx6uQQ0EjUwuC-Dhv7I3wMsVxyEKejv' +
-                  'jMLbfWJ6iu4-UUx1r8_ZZUjDDXSB3OFXJQ3nPwRVFXZuRNhGScL' +
-                  'nftXz7mypRGxrapIQusym1K8hk9uy8_KYL2H2QNbyIqK9Vh9JhY' +
-                  '1rtkQNpv3ZerCUXEVGRiEXDqR_OHu4vUi1-FkONZZe2ov8dQ1mX' +
-                  'iHHdw',
-        expectations: function (test, res) {
-          expect(res).toEqual(false);
-        }
-      });
-
-      setupVerifyIdTokenTest({
-        title: 'rejects an invalid idToken due to expiration',
-        idToken: tokens.standardIdToken,
-        execute: function(test, opts) {
-          util.warpToDistantPast();
-          return test.oa.idToken.verify(opts.idToken, opts.verifyOpts)
-            .then(function(res) {
-              util.returnToPresent();
-              return res;
-            });
-        },
-        expectations: function (test, res) {
-          expect(res).toEqual(false);
-        }
-      });
-
-      setupVerifyIdTokenTest({
-        title: 'verifies an idToken that would be invalid, except ' +
-                'we\'re using the expirationTime option',
-        idToken: tokens.standardIdToken,
-        verifyOpts: {
-          expirationTime: 9999999999
-        },
-        execute: function(test, opts) {
-          util.warpToDistantPast();
-          return test.oa.idToken.verify(opts.idToken, opts.verifyOpts)
-            .then(function(res) {
-              util.returnToPresent();
-              return res;
-            });
-        },
-        expectations: function (test, res) {
-          expect(res).toEqual(true);
-        }
-      });
-
-      setupVerifyIdTokenTest({
-        title: 'verifies a valid idToken using single audience option',
-        idToken: tokens.standardIdToken,
-        verifyOpts: {
-          audience: 'NPSfOkH5eZrTy8PMDlvx'
-        },
-        expectations: function (test, res) {
-          expect(res).toEqual(true);
-        }
-      });
-
-      setupVerifyIdTokenTest({
-        title: 'rejects an invalid idToken using single audience option',
-        idToken: tokens.standardIdToken,
-        verifyOpts: {
-          audience: 'invalid'
-        },
-        expectations: function (test, res) {
-          expect(res).toEqual(false);
-        }
-      });
-
-      setupVerifyIdTokenTest({
-        title: 'verifies a valid idToken using multiple audience option (all valid)',
-        idToken: tokens.standardIdToken,
-        verifyOpts: {
-          audience: ['NPSfOkH5eZrTy8PMDlvx', 'NPSfOkH5eZrTy8PMDlvx']
```

Comment:
I was looking at the tests for `token.verify`, and I'm not seeing any assertions about multiple audiences.  Should we carry these tests over there?

Reply:
Picking up this (and a few others discussed offline) in a separate PR.